src/status_im/ui/screens/desktop/main/chat/views.cljs

```diff
@@ -10,7 +10,10 @@
             [status-im.utils.identicon :as identicon]
             [status-im.utils.datetime :as time]
             [status-im.ui.components.styles :as styles]
-            [status-im.ui.components.react :as react]))
+            [status-im.ui.components.react :as react]
+            [status-im.ui.components.colors :as colors]
+            [status-im.chat.views.message.datemark :as message-datemark]
```

Comment:
we don't use `-` in aliases, better to use `:as message.datemark`

Reply:
@flexsurfer, could you please double confirm this one? I made a search and see a lot of `-` sign in aliases in a code. Like here - https://github.com/status-im/status-react/blob/develop/src/status_im/chat/screen.cljs
`toolbar-content`, `message-options`, `list-selection`, `message-datemark`, etc...

Reply:
yes i can confirm this one, your link is target old unrefactored code, http://tonsky.me/blog/readable-clojure/#use-consistent-unique-namespace-aliases 
![image](https://user-images.githubusercontent.com/11790366/41582916-969a813a-73ab-11e8-89d0-946a246d13cd.png)